### ISSUE TITLE
[bug] 1.10.2 Weird visual artifacts

### ISSUE THREAD

**Issue by winsrp** [darkevilmac/archimedes-ships-plus#118]
Hi, 

I have my computer set with 2 mods, your davincis vessels and malisis doors. Using malisis door over forge alone works fine but when I put your mod on the following occurs.

When I put a hi-tech door, the door gets placed fine, but when I break it, a magenta a blank version of the door stays in place and cannot be broken, removed or anything, If I take out your mod it works just fine. Can you please give it a look. I love this mod, but this is really weird. plz.

Thanks.

**Comment by Divineaspect** [comment]
This is all Malisis Doors, does it with a LOT of mods.

**Comment by exaskye** [comment]
Reopening following discussion with Ordinastie, the author of MalisisDoors. This is a MovingWorld issue.

**Comment by exaskye** [comment]
PR for 1.11: elytra/MovingWorld#31
PR for 1.10: elytra/MovingWorld#30

@Darkevilmac, please review and pull (yes, I just pinged you four times)

**Comment by zandercam7** [comment]
I am having the exact same issue as winsrp.  I placed a Hi-Tech Door in the middle of my house's floor (simply to check it out) and went to delete it, because obviously I don't need a Hi-Tech Door in the middle of my floor...  When I broke the door, it turned into a purple and black shadow.  This purple/black shadow did not go away when I deleted my server world file and re-uploaded the world file from the one saved prior to the door's creation.  It is still there, and I cannot get rid of it.  I have tried to destroy the door by using worldedit to build through it, but when I destroy the blocks, the black/purple shadow is still inside.  How do I fix this?
